doc/source/report/ultradian.rst

```diff
@@ -1,92 +1,66 @@
 .. _ultradian:
 
-Ultradian Analysis
-==================
+Ultradian and Circadian Analysis
+================================
 
 Statement of problem
 --------------------
 
-Ultradian rhythm is widely observed in mammalian behavioral patterns.
-Ultradian analysis aims to find the time-specific patterns in behavioral
-records, without specifying the length of cycle in advance (but need to be
-within 1 hour to 1 day). Typical ultradian period for rats includes 4, 12 and
-24 hours. For example, we expect rats to be inactive in the nighttime.
-Ingestions and movements mostly happened in the daytime. It would be
-informative to study the ultradian cycle of the behavior of mouse and we need
-to answer the following questions for this study:
-
-What is the variable of interest for the periodic patterns? 
-
-- Summary of activity: Food and water ingestion, distance traveled , movement
-  intensity, AS probability.
-
-- Spatial variable: Spatially discrete the data to cells each with its primary
-  functions such as food cell, water cell, etc. Examine ultradian cycle of the
-  spatial probability densities of the occupancy time in each cells.
-
-How to subset the data?
-
-- Basic subset: 16 strains.
-
-- Strains may not be the primary influence for the variation of ultradian
-  rhythms. We may look into the cycle for each mouse and detect the most
-  important factors influencing the ultradian rhythms.
-
-How to choose the frequency or period ?
-
-- The Lomb-Scargle (LS) periodogram spectral analysis technique, a widely used
-  tool in period detection and frequency analysis.
-
-What is the connection with other subprojects?
-
-- Ultradian rhythms could be treated as one feature for clustering the 16
-  strains. We may also subset the data using the results of the cluster and
-  analysis the rhythm similarities and differences across clusters.
+Ultradian and circadian rhythm are widely observed in mammalian
+behavioral patterns. The ultradian analysis aims to
+find the time-specific patterns in behavioral
+records, without specifying the length of the cycle in advance (but need to be
+within 1 hour to 1 day). The typical ultradian period includes 8 and 12 hours.
+The circadian rhythm refers to a roughly 24-hour cycle.
+For example, we expect the rats to be inactive in the nighttime.
+And ingestions and movements mostly happened in the daytime.
```

Comment:
Hey guys! Jarrod has asked me to review some of the wording of the projects so I'll make some comments if I see anything. The report looks great and it's just minor changes, I hope it's not imposing!
-Alanna

"For example, we expect the rats to be inactive in the nighttime 
and ingestions and movements mostly happened in the daytime."